app_server/app/controllers/StreamController.js

```diff
@@ -74,6 +75,13 @@ Class.registerRoutes = function() {
                        auth: {scope: Authenticator.SCOPE.ADMIN.ADMINS}
                      },
                      handler: this.deleteStream});
+
+  this.server.route({method: 'GET', path: '/statistics',
+                     config: {
+                       validate: streamsStatsValidator,
+                       auth: {scope: Authenticator.SCOPE.ADMIN.ADMINS}
+                     },
```

Comment:
Can we just do `/stats` for the sake of brevity?
Also, it's `ADMIN.DEFAULT` for all admins; `.ADMINS` is for admins with CRUD access to the admins' data.
`ADMIN.METRICS` may be more suitable for stats, though.